docs/ref/reference.rst

```diff
@@ -119,6 +120,11 @@ commands are dealing with the monitor:
      You may use the `--nodename` command line option to bypass the whole
      DNS lookup based process and force the local node name to a fixed
      value.
+     
+     The ``--auth`` option allows setting up authentication method to be used
+     for connections from data nodes with ``autoctl_node`` user.
+     If this option to be used, please make sure password is manually set on
```

Comment:
I think this should be written “If this option is used”, or maybe “is to be used”, though I prefer the first variant.

Reply:
ok